near-state-indexer/src/main.rs

```diff
@@ -70,14 +70,14 @@ async fn handle_streamer_message(
         &indexer_config,
     );
 
-    let published_streamer_message_feature =
-        utils::publish_streamer_message("final_block", &streamer_message, redis_client);
+    let update_block_streamer_message_feature =
```

Comment:
I believe you meant fUture
```suggestion
    let update_block_streamer_message_future =
```